src/mscorlib/src/System/Threading/Tasks/Task.cs

```diff
@@ -2339,21 +2309,6 @@ void IThreadPoolWorkItem.ExecuteWorkItem()
         }
 
         /// <summary>
-        /// The ThreadPool calls this if a ThreadAbortException is thrown while trying to execute this workitem.  This may occur
-        /// before Task would otherwise be able to observe it.  
-        /// </summary>
-        void IThreadPoolWorkItem.MarkAborted(ThreadAbortException tae)
```

Comment:
How do you see enabling sharing with CoreRT? For `Task.cs` (eg) it has all the TAE code commented out. I was hoping to make this shareable with this change.